development/docs-style-guide.md

```diff
@@ -45,7 +42,7 @@ represents.
   <tr><td>Create <code>medic/translations/messages-en.properties</code> file.</td><td>Create medic/translations/messages-en.properties file.</td></tr>
 </table>
 
-### International standard for punctuation inside quotes
+### US standard for punctuation inside quotes
```

Comment:
I think the recommendation is to not use the American standard here, but use the British standard so that the punctuation is always outside of the quotes (unless the punction itself is part of the quote).

```suggestion
### British standard for punctuation inside quotes
```

Reply:
Oh you're right, I misread the order